Modules/itertoolsmodule.c

```diff
@@ -546,50 +573,26 @@ static PyMethodDef groupby_methods[] = {
     {NULL,              NULL}           /* sentinel */
 };
 
-static PyTypeObject groupby_type = {
```

Comment:
Please stay focus on your core task and don't make these gratuitous stylistic changes. That just interferes with maintenance of the module making it more difficult to backport bug fixes.  It also means I have to deal with merge conflicts in patches that I current have in development.

Reply:
This is not a stylistic change; please look again. The original lines represented a static `PyTypeObject` struct. The new lines represent an array of `PyType_Slot` structs. How else would you suggest I do this particular change?

Reply:
Ditto for the `_grouper` type; these changes are needed in order to convert a _static_ type to a _heap_ type.

Reply:
Please leave lines 550-590 as-is.   Unnecessarily changing the struct initialization style makes these PRs very large so that it is hard to see what is substantively changing.

Reply:
Raymond, I urge you to read PEP-687 and PEP-630[^1].

Please understand this:

In order to isolate an extension module, we must convert _all_ static types to heap types. This means we must **remove** all static type structs and convert them to `PyType_Slot` arrays; these are **fundamentally different data types** and need **fundamentally different initialisation**. We _can not_ leave lines 550-590 as they are; that will block the effort of isolating `itertools`.

Raymond, please understand that **static types** and **heap types** are created using different data structures and different C APIs. I repeat: we _can not_ leave lines 550-590 as they are.

[^1]: Both PEPs are accepted by the SC.

Reply:
I'm marking this as resolved, since the requested change does not make sense. I've explained the reason for why these lines must be changed multiple times. There is nothing more to debate here.